### ISSUE TITLE
Pool/Lane containment bug.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [luisvt/oryx-editor#5]
```
Ist: Man legt einen Pool an, resized diesen und fügt eine Lane ein. Effekt:
Pool wird resized

Soll: Pool wird nicht resized 
```

Original issue reported on code.google.com by `martin.c...@googlemail.com` on 5 Nov 2007 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
we have to think about how resizing a shape object with containing shape 
objects will
manipulate the cotaining shapes' position and size. A solution would also solve 
this
issue.
```

Original comment by `NicoPete...@gmail.com` on 16 Apr 2008 at 3:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `W.Tsches...@gmail.com` on 22 Apr 2008 at 11:48

**Comment by GoogleCodeExporter** [comment]
```
will be fixed via issue #70.
```

Original comment by `NicoPete...@gmail.com` on 22 Apr 2008 at 2:23
- Changed state: **Duplicate**
